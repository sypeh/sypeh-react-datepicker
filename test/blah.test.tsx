import React from 'react'
import { configure } from 'enzyme'
import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
import renderer from 'react-test-renderer';

import { SyButton } from '../src/components/button/SyButton'

configure({ adapter: new Adapter() })

describe('SyButton', () => {
  it('renders basic wrapper correctly', () => {
    const component = renderer.create(<SyButton/>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot()
  })
})