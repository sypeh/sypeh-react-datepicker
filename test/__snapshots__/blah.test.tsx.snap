// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SyButton renders basic wrapper correctly 1`] = `
<button
  className="btn btn-primary"
  onClick={[Function]}
  type="button"
/>
`;
